aries-site/src/examples/templates/FilterControls/filterTypes/FilterCheckBoxGroup.js

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, CheckBoxGroup, FormField } from 'grommet';
+
+import { useFilters } from '..';
+
+export const FilterCheckBoxGroup = ({ attr }) => {
+  const { data, filters, setFilters, getFilterOptions } = useFilters();
+  const { property, label, inputProps, sort = true, convertToString } = attr;
```

Comment:
Not sure if I'm following why and where these are coming from? Are they from the styled label?

Reply:
Good question. If I understand it correctly, this could/should be documented/commented.

A caller's data might have a large number of columns, however, from a filtering perspective they might want to limit/focus which of those columns should be presented as a filter.

The proposal is for the caller to specify which data attributes in their data set by which they'd like to filter. They would specify the filter configuration and pass it to a `filters` prop: `<FilterControls filters={filtersConfig} />`. The configuration properties would be:
- property = field/column of interest in the data set
- label = user facing label to present as the filter's FormField label
- filterType = type of input component to use as the filter
- inputProps = any props supported by the component specified by the filterType
- sort = boolean for whether the filter options should be sorted (e.g. options in a checkboxgroup or select)
- convertToString = present numeric values in a CheckBoxGroup (perhaps should be named numericToString)